packages/secretlint/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     /* Basic Options */
     "module": "commonjs",
-    "moduleResolution": "node",
+    "moduleResolution": "NodeNext",
```

Comment:
read-pkg is pure ESM package.
current `secretlint` package is CJS package.
so, it need to use dynamic import for loading read-pkg.